editions/tw5.com/tiddlywiki.info

```diff
@@ -18,7 +18,8 @@
 		"tiddlywiki/tight",
 		"tiddlywiki/heavier",
 		"tiddlywiki/tight-heavier",
-		"tiddlywiki/readonly"
+		"tiddlywiki/readonly",
+		"tiddlywiki/example"
```

Comment:
What is tiddlywiki/example used for

Is it there for testing only, or will it stay?

Reply:
Should this theme be renamed? eg: "resizers" or something similar?